documentation.html

```diff
@@ -74,4 +74,14 @@ <h2>Videos</h2>
         <li><a href="https://www.youtube.com/results?search_query=openQA">Various videos on YouTube</a></li>
     </p>
   </div>
+    
+  <div class="col-md-12">
+    <h2>Blogs and external resources</h2>
+    <p>
+    If you are contributing to openQA by writing posts, videos, podcasts, ... consider adding your content to the list of external resources below:
```

Comment:
```suggestion
    If you are contributing to openQA by writing posts, videos, podcasts, … consider adding your content to the list of external resources below:
```

Reply:
I would generally phrase in a way that you can read out loud. And semantically "content" is not what you're adding here:

```suggestion
    If you are contributing to openQA by writing posts, videos or podcasts consider adding a link to the list of external resources below:
```

Reply:
I agree with @kalikiana and pushed his suggestion.